Kitodo/src/main/resources/mapping.json

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
Indent looks strange: between 1, 2 and 4 space indent. Could this be uniformed to one identical indent (Java code use 4 space indent).